From 499fe07d508517d792832bae5700dca8fa761e9d Mon Sep 17 00:00:00 2001
From: "qipeng.zha" <qipeng.zha@intel.com>
Date: Tue, 4 Nov 2014 07:12:44 +0800
Subject: [PATCH 1/1] Touch: fix conflicting irq flag issue

Cherrytrail GPIO doesn't support stacked trigger mode flags.
Setting default to IRQF_TRIGGER_LOW to ensure irqs are triggered.

Change-Id: I7b5879eda4998e4506eb7385a50eb03737f5bea1
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-7309
Signed-off-by: qipeng.zha <qipeng.zha@intel.com>
---
 drivers/input/touchscreen/atmel_mxt_ts.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/input/touchscreen/atmel_mxt_ts.c b/drivers/input/touchscreen/atmel_mxt_ts.c
index 4ff283b03c79..4bffe7f7250b 100644
--- a/drivers/input/touchscreen/atmel_mxt_ts.c
+++ b/drivers/input/touchscreen/atmel_mxt_ts.c
@@ -3643,7 +3643,7 @@ static int mxt_probe(struct i2c_client *client,
 		data->pdata = pdata;
 
 		/* Set default parameters */
-		pdata->irqflags = IRQF_TRIGGER_LOW | IRQF_TRIGGER_FALLING;
+		pdata->irqflags = IRQF_TRIGGER_LOW;
 
 		gpio = devm_gpiod_get_index(&client->dev, "atml_gpio_rst", 0);
 		if (!IS_ERR(gpio)) {
-- 
1.7.9.5

